### DISCUSSION TITLE
Undefined control sequence. <argument> \tikzexternaldisable 这个问题怎么解决呢？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#173] [Q&A] [answered]
我今天只在论文中加了表格和pdf图片，单独运行都是正常的。但是放入昨天的文件中运行，提示：
```
Undefined control sequence. <argument> \tikzexternaldisable 
```
我今天加的内容没有使用`tikz`的，倒是之前有使用`tikz`的，但是不加今天的内容昨天是没有问题的。这会是什么原因呢？
谢谢

**Comment by sikouhjw** [comment, marked as answer]
给 MWE

**Reply by earthmoon** [reply]
给不出哎，单独部分都是对的。

**Reply by earthmoon** [reply]
> 给 MWE

谢谢你！可以这样解决：`\usetikzlibrary{external}`添加到`\begin{document}`之前就好了。
